### PR TITLE
Add more fields to the JSON generator, and add a method to return the non-serialized JSON object

### PR DESCRIPTION
**Pull request by netroy**

### CODE REVIEW

**Comment by netroy** [comment]
@verhovsky any opinions on this?

File: src/generators/json.ts [hunk 1]
```diff
@@ -19,6 +19,9 @@ export const supportedArgs = new Set([
   "location-trusted",
   "no-location-trusted",
 
+  "output",
+  "include",
+
```
Comment (verhovsky) [inline src/generators/json.ts:24]:
```suggestion
  "max-redirs",

  "output",
  "include",

  "proxy",

```

**Comment by verhovsky** [comment]
Thanks

**Comment by netroy** [comment]
@verhovsky thanks for merging this. 🙏🏽
is there an ETA on when the next release will be?

**Comment by netroy** [comment]
can we please have this released to npm 🙏🏽

**Comment by netroy** [comment]
Thanks @verhovsky 💟

**Pull request merged**